docs/src/views/FAQ.tsx

```diff
@@ -111,6 +111,45 @@ export default () => (
       }
     />
 
+    <Question>
+      What's the difference between {code('Flex')}, {code('Grid')}, {code('Layout')}, {code('Box')}{' '}
+      and {code('Segment')} components?
+    </Question>
+    <Answer
+      content={
+        <div>
+          <p>
+            {code('Flex')}, {code('Grid')} and {code('Layout')} components handle layout aspects.
+            The {code('Flex')} component is for laying out items in one direction, while the{' '}
+            {code('Grid')} component is made for two dimensional layouts. Visit the{' '}
+            {link('Layout guide', '/layout')} page for a detailed comparison between {code('Flex')}
+            and {code('Grid')}.
+          </p>
+          <p>
+            The {code('Layout')} component is now deprecated. Its purpose was arrangement of the
+            content of a component. {code('Flex')} or {code('Grid')} component should be used
+            instead of the {code('Layout')}
+            component.
+          </p>
+          <p>
+            {code('Box')} is a utility component that is often used by Stardust to implement
+            higher-level components. By default, it renders styled {code('div')} element.
+          </p>
+          <p>
+            The cases when client might want to use it are very exceptional, and all are about
+            applying custom styles to rendered {code('div')} element. However, more robust approach
+            that won't break theming consistency is to instead create a custom component and define
+            related styles as part of theme{' '}
+            {link('Integrate Custom Components', '/integrate-custom-components')}.
```

Comment:
The ending of the paragraph is not good. Let's end it with something like: 

> Visit the `Integrate Custom Components` page to see how this can be done

Or something similar, at least to give transition of what the link is about.